README.md

```diff
@@ -1 +1,288 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project">Hello Microverse</a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse]** is a simple Html and Css program
+
```

Comment:
- [ ] your readme is basically the same as the [microverse template readme](https://github.com/microverseinc/readme-template) but this shouldn't be the case. you are required to customize this readme to reflect the current project which you worked on. please i will suggest you kindly follow the guideline which i will propose to you in setting up your readme before submitting another review request. :grin: 
* [ ]  For the project requirements, kindly ensure that your README file is descriptive and professional. And the required template to be used here is the **[Microverse README template](https://github.com/microverseinc/readme-template)**, so ensure that:
* All main headers from the [README template](https://github.com/microverseinc/readme-template) are present in a your repo.
* The only optional section is `FAQ`
* you delete the `live demo` section if the you decide not to deploy the application.
* Make sure the you have added the section `Future Features`, including 1-3 potential features to be added.
* you remove the section `description`, i.e.` "> Describe the tech stack and include only the relevant sections that apply to your project."` should be removed and filled in with their custom content.
* you remove template logo. Optionally, the you can replace it with your own logo you they have one.
* No broken link or image

> - you can checkout this [student's readme](https://github.com/M-Anwar-Hussaini/HelloWorld-Microverse/blob/master/README.md) for inspiration on how to customise some things after following the guidelines provided above.